packages/@coorpacademy-components/src/molecule/login-alert/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {NovaSolidInterfaceFeedbackInterfaceAlertTriangle as AlertIcon} from '@coorpacademy/nova-icons';
+import Provider from '../../atom/provider';
+import {container, iconContainer, content, icon} from './style.css';
+
+const LoginAlert = ({body}, context) => (
+  <div className={container}>
+    <div
+      className={content}
+      // eslint-disable-next-line react/no-danger
+      dangerouslySetInnerHTML={{__html: body}}
+    />
+    <div className={iconContainer}>
+      <AlertIcon height={16} className={icon} />
```

Comment:
```suggestion
      <AlertIcon className={icon} />
```

First, your class icon does not exist in the css file
Second, since you want to use css, just define the height in the css pls

Reply:
Could you also import style rather than distructuring it?